webapp/src/Entity/Contest.php

```diff
@@ -341,6 +387,7 @@ public function __construct()
         $this->submissions      = new ArrayCollection();
         $this->internal_errors  = new ArrayCollection();
         $this->team_categories  = new ArrayCollection();
+        $this->awardsCategories = new ArrayCollection();
```

Comment:
Not for you but we should really make our variables consistent in casing...

Reply:
Fine, maybe `awards_categories`?

Reply:
For now sure. We will check later to make it clean, we have some (like `removedIntervals`) that are ugly still.